site/docs/spark-configuration.md

```diff
@@ -155,6 +155,7 @@ spark.read
 | file-open-cost  | As per table property | Overrides this table's read.split.open-file-cost                                          |
 | vectorization-enabled  | As per table property | Overrides this table's read.parquet.vectorization.enabled                                          |
 | batch-size  | As per table property | Overrides this table's read.parquet.vectorization.batch-size                                          |
+| stream-from-timestamp | Long.MIN_VALUE | Timestamp in milliseconds, start streaming this table from the first snapshot that occurs at or after this timestamp |
```

Comment:
I think that the default is not set, so the default behavior is not to stream from a timestamp but to stream from the starting timestamp.

Reply:
You may also want to update this to "first known ancestor snapshot" and add a note:

```
!!! Note
    If `stream-from-timestamp` is before the oldest ancestor snapshot in the table, the oldest ancestor will be used.
```

Reply:
Thanks, for the inputs @rdblue.

The default value `Long.MIN_VALUE` is set here: https://github.com/apache/iceberg/blob/master/spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/SparkReadConf.java#L215-L220

I have updated the wordings to "first known ancestor snapshot". 

I have added the note about the default behavior. Do let me know if I should keep the note in the default column & remove `Long.MIN_VALUE`.

Reply:
Please remove `Long.MIN_VALUE`. While that's in the code, we want to document behavior, not the specific implementation.

Reply:
Thanks for the clarification, @rdblue. I have pushed the change.